cif_pow.dic

```diff
@@ -5829,6 +5969,51 @@ save_PD_PHASE_MASS
          '_pd_phase_mass.diffractogram_id'
          '_pd_phase_mass.phase_id'
 
+    loop_
+      _description_example.case
+      _description_example.detail
+;
+         loop_
+         _pd_phase_mass.diffractogram_id
+         _pd_phase_mass.phase_id
+         _pd_phase_mass.percent
+         _pd_phase_mass.percent_su
+         A_DIFFRACTOGRAM   PHASE_1   45.45   0.42
+         A_DIFFRACTOGRAM   PHASE_2   37.42   0.63
+         A_DIFFRACTOGRAM   PHASE_3   17.13   0.53
+;
+;
+         Tabulation of quantitative phase analysis data. The phase masses,
+         expressed as percentages of the entire specimen are given, along with
+         their associated standard uncertainties. The values are associated with
+         one diffractogram and three phases.
+
+         The method of presentation given here would be associated with a
+         "summary" data block; one that doesn't contain any phase- or
+         diffractogram-specific information, but summarises results.
```

Comment:
It should be noted that "Summary" data blocks will have `_audit.schema` `Custom`, and this should be included in the example.

Reply:
Roger. I need to read up on how `_audit.*` works.

Reply:
Question: This example has all `Loop` category datanames in a loop. How is that a custom schema?

Reply:
Ah yes. Because a child data name of a `Set` category key is looped, that makes it Custom. The reason being that a legacy program would get confused if, e.g., atom site parameters for more than one phase were looped in a single data block. We may need to improve the description of how `_audit.schema` works.

Reply:
Then pdCIF is going to have custom schema everywhere.

From the PD_PREF_ORIENT_SPHERICAL_HARMONICS example:

```
_pd_phase.id   PHASE_A

         loop_
         _pd_pref_orient_spherical_harmonics.id
         _pd_pref_orient_spherical_harmonics.diffractogram_id
         _pd_pref_orient_spherical_harmonics.y_i
         _pd_pref_orient_spherical_harmonics.y_j
         _pd_pref_orient_spherical_harmonics.c_ij
         _pd_pref_orient_spherical_harmonics.c_ij_su
         a   DIFFPAT_A   0   0   1.0     .
         b   DIFFPAT_A   2   0   1.538   0.013
         c   DIFFPAT_A   4   0   0.913   0.016
         d   DIFFPAT_A   4  -3  -0.166   0.010
```

Reply:
Well, in that example, `_pd_pref_orient_spherical_harmonics.diffractogram_id` could just be left out and `_pd_diffractogram.id` provided as a single value elsewhere in the block.

The even better description for conforming to the `Default` schema is that only a single value of any child data name of a `Set` category key should be provided in any data block.